Src/Sarif.PatternMatcher.Cli/StressOptions.cs

```diff
@@ -9,6 +9,14 @@
 
 namespace Microsoft.CodeAnalysis.Sarif.PatternMatcher.Cli
 {
+    public enum StressScenario
+    {
+        Statelessness = 0,
+        RE2Timing,
+        SingleThreadedTelemetry,
```

Comment:
Earlier, there was a single `FileSizeVsTime` mode. I have split the scenarios to reflect what we want to measure:

1. Single threaded analysis: where the objective is to analyze analysis rate per core (ms/KB), and to measure the relative effect of analysis time as a function of file size.
2. Multiple threaded analysis: where the object is to measure the overall CPU/mem utilization.